README.md

```diff
@@ -76,8 +77,7 @@ For further information or help, don't hesitate to get in touch on the [Slack `#
 
 ## Citation
 
-<!-- TODO nf-core: Add citation for pipeline after first release. Uncomment lines below and update Zenodo doi. -->
-<!-- If you use  nf-core/mag for your analysis, please cite it using the following doi: [10.5281/zenodo.XXXXXX](https://doi.org/10.5281/zenodo.XXXXXX) -->
+If you use nf-core/mag for your analysis, please cite it using the following doi: [10.5281/zenodo.4068515](https://doi.org/10.5281/zenodo.4068515)
```

Comment:
```suggestion
If you use nf-core/mag for your analysis, please cite it using the following doi: [10.5281/zenodo.3589527](https://doi.org/10.5281/zenodo.3589527)
```

Reply:
Thanks @MaxUlysse , but I thought one would want here the DOI for the latest release, and not the generic one for all releases? Because this would not be resolved for the displayed DOI. Only when clicking on the link the user could figure out the DOI for the specific version.

Reply:
.. ok, maybe not for the latest, but for the actual release, as otherwise the READMEs for older version would be wrong. But still, if someone uses the pipeline, wouldn't it be encouraged to cite the actual version?

Reply:
You're technically right that this will always resolve to the latest version, e.g. if a user uses a version 1.0.0 at some point, this URL will point them to 1.1.0 (or whatever is the latest release for the pipeline) - which is going to be wrong at some point. However, as the DOI / badge above also links to an overview page that makes it easy via the navigation to get the version the user wants to obtain, I think its fine to use the latest url everywhere in order to make your life easier ;-)